src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php

```diff
@@ -44,7 +44,7 @@ class FileTree extends AbstractWidget
      *
      * @var string
      */
-    private $subTemplate = 'widget_filetree';
+    protected $subTemplate = 'widget_filetree';
```

Comment:
Why did you change this?
IMO this is wrong, as the value can be modified via getter and setter and no one should manipulate it directly.
Can you explain to me your reasoning?